src/components/tiptap/MenuBar.tsx

```diff
@@ -0,0 +1,165 @@
+
+import React, { Fragment } from 'react'
+import MenuItem from './MenuItem'
+
+type EditorProps = {
+  editor: any; 
+};
+type MenuItemProps =  {
+  icon: JSX.Element;
+  title: string;
+  action: () => void;
+  isActive: (() => boolean) | null | undefined;
```

Comment:
❓ 
why the type could be null or undefined?

Reply:
i deleted this because i don't need it anymore